### ISSUE TITLE
WASM support example

### ISSUE THREAD

**Issue by Luni-4** [ggand0/viewskater#7]
First, thanks a lot for `viewskater`, I really like this software!

It would be fantastic whether `viewskater` might have a `wasm` example which would show how to integrate this software into a webpage. This would allow a wide cross-platform coverage, since many systems, other than the already supported ones, would make use of it.

**Comment by ggand0** [comment]
Hi thanks for submitting this issue. It’s possible to build an Iced app for WASM, but this app relies on tokio's async operations to dynamically load images into cache from the file system, which isn’t compatible with WASM. In the future, I could explore supporting this by making this part download images from the web for example.

**Comment by Luni-4** [comment]
Thanks for your explanation! The possibility to download images from the web would be great!

Just to better figure out the problem, at the moment, isn't it possible to create a web page with `viewskater` integrated in it in any way?

**Comment by ggand0** [comment]
Right, it's not possible to integrate this app into a web page at the moment.
